ansible/group_vars/all/alaska

```diff
@@ -116,5 +116,8 @@ cluster_admins:
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDCLva2hRoqcA77XoK0t7aPmip4CmxM6SdsnJdbN4i0l72Rf6HMNH2ZR5p+eZPxaxiJgmtQ5MpquNt775lFSGVIV7oyAKDdYuLA/8KXoKje4EsCMfA9m6Eg8urjrPprelTAl7xsem1eH9c/Jk6TFbJSyMrY7kvgNo1KZ+D24F5cCSSRncy0s5GoPWEJfdnSHdjvxc97vhSPY9KwsaMdbpj+u+1WlUDeoLjMsWLPDsqnIHLwYN74f9jfTvsoQxw9Wq3dmzRvy8sQkJ1OoAeuuM4B4MKel7U6QqrhMYyIUG2pcSBBICtT2Q5V4RhnBIqQrL8w1efq+TeD/ETSuQgFBiq1 sclt100@ilab-gate.ilab.cluster"
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCoHB+D6iI859Fq4Wg3iWDPf+miX+nXDx8Y8F20C5wMa7uHFYWWp0gfhMslI3ntWvlPf9J7C8AQz/kzPeuvM9oEZUkBuH6oYtYrJ9nA6+IJqT/xbQGUy56hkVVFH8Etff9CzseczzhLPOd/CEj4irLdi4aoj/5V8tqvdAbnOGwixPRPmVKVvEVpFiAlvRi+0MCaWHF2lF4BVWJsRKKGf79DFOrhfI9+BefuD9essaCOuFfhkHSxngBIjicYJKSedMFoWaOSY35820P/6RnmbKjH/mMQ0yuEr7SgLZ3gdTN2crrDj0q8+yz5MHfuxPdtj9dOYb8yP8MahbOl91qIV5BL John@Lenovo-PC"
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDT7FfKdhVmT/HturOK6sLRKcqFYDnfbKWVE8Lv+KntJCBN0pGk5PECCmmA7/5JzAzfba4nmMpI20byJ0VCphzTbYFHtsZNd6Ci4NIKb5GrxP1f9RN3BYlmm1rJ5OgoXjvg0VpUiUnkLsVYkHfzUTx6tDHj8zV2DQYWGg2BtZk8YSEnigK6ppZcBuRwhY1wmKEodeXus6vik20v+PGDAlSmSlhHsYUOe8/GNhs6fZ9a9XEVHw/pPxYa3HmNGgUiJYOxSA0DLq8HzxQDoSgpMMeG2bD36DsHSnnbLNUtH3+DmveX2FxWWHRKG4IgW5O0UdLcGzrakZ6OFi/bUL0bBge7 bharat@dev-director.dev-cluster.hpc.cam.ac.uk"
+    - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCx6j0X18mPQ6YZSXrO5ZamYH6szsPhHzWxAvATYy3Lj6GEBVuVgM6XAhIfEocBhPgwW1tmA/1N12YpNeBGZwRjtjgCMdnq8rks5x8FI+2Jfvkv5a/XWI86t+LsmQz7gRl/9WwokOHUAKoPJMxEWztx7OHW8OlLU9j9iWe5TEVzcg4ra/9Se9X7Zut4WYnl680Z/BW1nV7O/lgUpimgtsY6cWvX+OHLhc/pPXy3j1vrxA/oRKMxJLwD2b3+az05J4zg8D/HA+pCi4qPvAk9wwSMAkacLS/yy5cGdDrtlWiM+dqFWwI5ypxi83KBekLUql28Yej9UMxLw350diFJkoC7 hpcszum1@ilab-gate.ilab.cluster"
+    - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCUOhekNNoWCVzdyks2ROdW4gjDxKxPXCMc0VetY08i/WkZPFFUZZOhAN13EVzNh2aVtRYIQSGheqbuIW5QvEIBJ4OqmF5Tjw/76JBQ2c8Vhq0wLboYnvscMD4fwyHbcezBZ67eDxz1P7jTJtoBrshm+vaIXP1PEsQqJOc6RN+a9F6Fqfjuxc+yjGkuwC0h61pdkVrvvqBHUa5cSyFo3A4BAkpzwq785uBSkC/jQH3CxzcbeomxG+1nxYM69qCDARxnPBkHPQHjHP5TxrpS/JS1/pPS6GlKDQ0NvZ+5Eo09QFt7kPoY+05TWwkwCOWXESMPKBFUmOowUgThCZyR/YW3 mark@mark-xps15"
     - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDtz8BtUJFQe1IMunfKA0pHSKOv8Xq5ehQ73aFAOQi/rb60X10MQvEdXlT5KOv0GfDB8VcJctVl0q7WIlzplrVfOMHn4L9Mgyko8uN/drT//6YtceGQdu+tAgogvYemLeieN5Ms7JJNUVBXE3h8wxYEY2O1h+ykeaB88IAG5iXzaP69WyAUdtun0ACu+8ixYESkzw18wVgVYcxKt8eehabHFsSP8eLPJOreTObIChcmR5Uq1D4sXl5nE7pmIwbAnd4bzVUdpjxYs/pjFPq4JujBF5HHBzFfyyaGVlq8GXFEtNEFMbYXLI9xQPXv/T+7NB+dLhdPGB/2w08sybyXkpqJ kbs@localhost.localdomain"
+    - "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDpwtPMNbEDSGlk6eBsG2L8ytYolSaVA+rRkR8pHGG0L+Lpvc8IfrQvx7T1rqlpfz2uiACy629SHHK/HUmNHm3YuQM6M1hQrGNz38COFrg5ls0KOvcSZnAEMXmVR9wtKhVF2CZ/+Fz3HVBZXznFD+S8av1R06O5cgoB1sRkUqsjJZi3d1dKV87a4Ive5RpHDnueAlcLaZsEDyKh3ve2Ez8B++9A3I93V2zq/wlVC7c3MbRrJsfYcCo91hPZPDSQvpJ7C+NJx+B9G/ZHXzh0ko8fTRQebq8pQexleY1p4Ylmxyzlm+2G3VpnsJDNnG1SdZhqbD9qnYi8qfmTLrnuuQAB" # Peter Wortmann
```

Comment:
Why is Peter using the centos user rather than his own account?

Reply:
He had his key added to the centos user previously as he wanted to try different configurations